setup.cake

```diff
@@ -11,7 +11,7 @@ BuildParameters.SetParameters(
     repositoryName: "Cake.Yeoman",
     appVeyorAccountName: "cakecontrib");
 
-BuildParameters.PrintParamters(Context);
+BuildParameters.PrintParameters(Context);
```

Comment:
@pascalberger aw man, I forgot this was in each setup.cake file.  I'll have to remember to put this into each repo.